pinot-core/src/test/java/org/apache/pinot/core/data/function/JsonFunctionsTest.java

```diff
@@ -137,4 +137,19 @@ public Object[][] jsonFunctionsDataProvider()
     });
     return inputs.toArray(new Object[0][]);
   }
+
+  @Test(description = "jsonFormat(Java null) should return Java null")
+  public void jsonFormatWithJavaNullReturnsJavaNull() {
+    GenericRow row = new GenericRow();
+    row.putValue("jsonMap", null);
+    testFunction("json_format(jsonMap)", Lists.newArrayList("jsonMap"), row, null);
```

Comment:
This test fails if executed in master. Instead of Java null, what was returned was the String "null"